src/Query/Builder.php

```diff
@@ -950,19 +926,6 @@ protected function compileWheres(): array
                 }
             }
 
-            // Convert id's.
```

Comment:
In the `Query\Builder`, we don't know the model. We cannot guess the type of the `_id` field. It's better not doing any transformation.